src/components/SensorPageHeader/withData.tsx

```diff
@@ -0,0 +1,204 @@
+            categoryId: mergedSensor.category_id,
+            integration:
+              mergedSensor.connection_type === "other"
+                ? undefined
+                : mergedSensor.connection_type,
+            ttnDeviceId: mergedSensor.external_id,
+            latitude: mergedSensor.latitude,
+            longitude: mergedSensor.longitude,
+          }}
+          onSubmit={data => {
+            updateSensor({
+              id: initialSensor.id,
+              external_id:
+                data.ttnDeviceId || mergedSensor.external_id || undefined,
+              name: data.name,
+              description: data.description,
+              connection_type: data.integration,
+              longitude: data.longitude,
+              latitude: data.latitude,
+              category_id: data.categoryId,
+              icon_id: data.symbolId,
+              user_id: authenticatedUser?.id,
+            })
+              .then(() => setShowEditSuccessAlert(true))
+              .finally(() => setEditModalIsOpen(false));
+          }}
+          onDelete={() => setDeletionConfirmationIsOpened(true)}
+        />
+      )}
+      <SensorPageHeader
+        id={initialSensor.id}
+        name={mergedSensor.name || ""}
+        description={mergedSensor.description || ""}
+        category={userCategory || { id: 1, name: "C02", description: "" }}
+        symbol={mergedSensor.icon_id || fallbackIconId.current}
+        geocoordinates={{
+          latitude: mergedSensor.latitude || 52.4961458,
+          longitude: mergedSensor.longitude || 13.4335723,
+        }}
+        author={{
+          username: initialSensor.user.name || "anonymous",
+          displayName: initialSensor.user.display_name || "Anonymous",
+        }}
+        withEditButton={!!user && user.name === initialSensor.user?.name}
+        onEditButtonClick={() => {
+          setEditModalIsOpen(true);
+          setShowEditSuccessAlert(false);
+        }}
+      />
+      {deletionConfirmationIsOpened && (
+        <SmallModalOverlay
+          title={`Sensor löschen`}
+          footerContent={
+            <>
+              <Button
+                variant='dangerous'
+                onClick={() => {
+                  deleteSensor(initialSensor.id)
+                    .then(() =>
+                      router.push(
+                        user?.name ? `/accounts/${user.name}` : `/sensors`
+                      )
+                    )
+                    .finally(() => setEditModalIsOpen(false));
+                }}
+              >
+                Unwiderruflich löschen
+              </Button>
+              <Button
+                variant='secondary'
+                onClick={() => setDeletionConfirmationIsOpened(false)}
+              >
+                Abbrechen
+              </Button>
+            </>
+          }
+        >
+          <div className='prose max-w-none'>
+            <p>Du bist dabeit der Sensor „{mergedSensor.name}“ zu löschen.</p>
```

Comment:
Typos, should be: Du bist dabei, den Sensor [...] zu löschen.

Reply:
OMG all the typo... Sorry. Danke :D